Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -488,29 +518,44 @@ int PcapLiveDevice::startCaptureBlockingMode(OnPacketArrivesStopBlocking onPacke
 	m_cbOnPacketArrivesBlockingMode = onPacketArrives;
 	m_cbOnPacketArrivesBlockingModeUserCookie = userCookie;
 
-	long startTimeSec = 0, startTimeNSec = 0;
-	clockGetTime(startTimeSec, startTimeNSec);
-
-	long curTimeSec = 0;
+	auto startTime = std::chrono::steady_clock::now();
 
+	auto currentTime = startTime;
 	m_CaptureThreadStarted = true;
 	m_StopThread = false;
 
-	if (timeout <= 0)
```

Comment:
In the previous code we supported "no timeout" mode where we call `pcap_dispatch` forever until the user decides to stop capture. I think we should continue supporting it

Reply:
yes, the behavior is still the same. I did a refactoring to remove the duplicate `while()`, please check the new code.

from 
```
if(timeout) {
 while(stop) { pcap() }
else {
 while(stop) { pcap() }
}
```
to
```
while(stop){
  if{ pcap() }
  else{ pcap() } 
  
  if(timeout){ break;}
}
```

Reply:
you're right I missed that. Thank you for the explanation!